python/lsst/afw/geom/polygon/polygon.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import absolute_import, division, print_function
+
+from ._polygon import Polygon
+
+__all__ = []
```

Comment:
I don't know about SWIG. It's clearly appropriate for our code though I see I left off my usual comment, which I will add.